### DISCUSSION TITLE
Do you have speed limits for the transfer per file?

### DISCUSSION THREAD

**Discussion by TomHsiung** [iterate-ch/cyberduck#14656] [Q&A]
Other things constant, I transferred the very same file from the same server to the same client via Cyberduck and FileZilla respectively. The speed are about 33 MB/s vs 105 MB/s. Huge difference, hmmm. I asked openAI and,

chatGPT 3.5 said:
> Client limitations: The sFTP client may have limitations on the maximum bandwidth that a single transfer can use. This could be due to software settings or hardware limitations. In this case, the transfer may be throttled by the client and not reach the maximum bandwidth.

So is that true? I want full bandwidth usage.

**Comment by dkocher** [comment]
Refer to our [documentation](https://docs.cyberduck.io/cyberduck/transfer/#bandwidth) for configuration of bandwidth limits. By default it should not be limited. Enable [segmented downloads](https://docs.cyberduck.io/cyberduck/download/#segmented-downloads) to saturate available bandwidth for downloads.

**Reply by TomHsiung** [reply]
Hello, sir

I do enable the segmented downloads function. It just cannot reach or nearly reach the 1000 Mbps rate.
